hooks/gen_docs/__init__.py

```diff
@@ -2,7 +2,9 @@
 
 from collections.abc import Generator
 from enum import Enum
-from typing import Any
+from typing import Any, TypeVar
+
+_T = TypeVar("_T")
 
 
 class SuperEnum(Enum):
```

Comment:
should have `Generic[_T]`

Reply:
Ah, I think I got it. Is it correct now?